doc/api/util.md

```diff
@@ -10,6 +10,67 @@ module developers as well. It can be accessed using:
 const util = require('util');
 ```
 
+## util.callbackify(original)
+<!-- YAML
+added: REPLACEME
+-->
+
+* `original` {Function} An `async` function
+* Returns: {Function} a callback style function
+
+This method converts Promise returning API endpoints to ones that use callbacks.
+`callbackify` takes an `async` function (or a function that returns a Promise)
+and returns a function following the common Node.js callback style, i.e. taking
```

Comment:
The part of the sentence starting with "following the common Node.js" could probably just be "following the Node.js error first callback style."